files/setup.sh

```diff
@@ -70,13 +71,33 @@ Address=${IP_ADDRESS}/${NETMASK}
 Gateway=${GATEWAY}
 DNS=${DNS_SERVER}
 Domain=${DNS_DOMAIN}
+__CUSTOMIZE_PHOTON__
+    #########################
+    ### NTP Settings      ###
+    #########################
+    NTP_SERVER=$(echo "${NTP_SERVER_PROPERTY}" | awk -F 'oe:value="' '{print $2}' | awk -F '"' '{print $1}')
+
+    echo -e "\e[92mConfiguring NTP ..." > /dev/console
+    cat > /etc/systemd/timesyncd.conf << __CUSTOMIZE_PHOTON__
+
+[Match]
+Name=e*
+
+[Time]
+NTP=${NTP_SERVER}
+#FallbackNTP=time1.google.com time2.google.com time3.google.com time4.google.com
+#RootDistanceMaxSec=5
+#PollIntervalMinSec=32
+#PollIntervalMaxSec=2048
```

Comment:
Given these are comments, we probably don't need them as we're not using them?

Reply:
Thx for reviewing William. I agree with you and I´ll delete the comments.